.github/workflows/molecule_tests.yml

```diff
@@ -0,0 +1,37 @@
+---
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2021
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+name: Check Molecule Tests
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
```

Comment:
I think just `pull_request` is sufficient - we've generally not been repeating tests on `push`.